packages/app/src/cli/models/extensions/ui-specifications/post_purchase_extension.ts

```diff
@@ -8,7 +8,7 @@ const spec = createUIExtensionSpec({
   identifier: 'checkout_post_purchase',
   externalIdentifier: 'post_purchase_ui',
   externalName: 'Post-purchase UI',
-  surface: 'post_purchase',
+  surface: 'post-checkout',
```

Comment:
i'm removing the `surface` value from the specs in a WIP PR. 
We will only use the surface provided by the Specifications API, so that there are no confusions.